File: conditinals/time_of_day.c
//andrea lugo, time of day c
#include <stdio.h>
#include <time.h>

int main(void){
    //time since jan 1, 1970
    time_t seconds;

    seconds = time(NULL);
    printf("seconds since januari 1, 1970 = %ld\n", seconds);

    //current time
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    printf("current time and date is %s", asctime(timeinfo));

//curent hour
time_t now = time(NULL);
struct tm *tm_struct = localtime(&now);
int hour = tm_struct->tm_hour;
printf("%d\n", hour);

if (hour <12){
    printf("good morning\n");
}else if (hour <19 ){
    printf("good afternoon\n");
}else if (hour <24){
    printf("good evening\n");
}
return 0; 

}

File: conditinals/time_of_day.py
#andrea lugo, how to get the time of day python

import time 
#first instance of time in python
#print(time.gmtime())

#current time in second since gmtime
current = time.time()

#current time as were are used to seeing it
now = time.ctime(current)
print(now)

#get just the hour
local_time = time.localtime(current)
hour = local_time.tm_hour
minutes = local_time.tm_min
print(minutes)

if hour >7 and hour <8:
    print("good morning")
elif hour >11 and hour <12: 
    print("good afternoon")
elif hour >18 and hour <19:
    print("good evening")